lib/Makefile.include

```diff
@@ -23,6 +23,8 @@ ifneq ($(V),1)
 Q := @
 endif
 
+TOPDIR = $(SRCLIBDIR)/..
```

Comment:
I don't understand what this commit is still for?  It just adds another variable, but nothing that uses it?

Reply:
Yeah, you're right. Looks like it can just be removed.